android/src/ti/crashlytics/TitaniumCrashlyticsModule.java

```diff
@@ -50,4 +44,17 @@ public void setCrashlyticsCollectionEnabled(boolean crashlyticsCollectionEnabled
 	{
 		FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(crashlyticsCollectionEnabled);
 	}
+
+	@Kroll.onAppCreate
+	public static void onAppCreate(TiApplication app) {
+		FirebaseCrashlytics.getInstance().recordException(new Exception());
+	}
+
+
+	// @Override
+	// protected void initActivity(Activity activity) {
+	// 	super.initActivity(activity);
+	// 	Log.i("-----", "App init activity");
+	// 	FirebaseCrashlytics.getInstance().recordException(new Exception());
+	// }
```

Comment:
Can you remove the empty lines and commented out code? Then it should be good to go

Reply:
they are removed already